app/scripts/controllers/swaps.js

```diff
@@ -492,13 +495,21 @@ export default class SwapsController {
     return newQuotes;
   }
 
-  timedoutGasReturn(tradeTxParams) {
+  timedoutGasReturn(tradeTxParams, aggregator = '') {
     return new Promise((resolve) => {
       let gasTimedOut = false;
 
       const gasTimeout = setTimeout(() => {
         gasTimedOut = true;
-        resolve({ gasLimit: null, simulationFails: true });
+        captureException(new Error('timedoutGasReturn: timeout'), {
```

Comment:
It seems like this error represents a timeout, which is something that can happen in the normal operation of the wallet (e.g. in poor network conditions). We use Sentry just for capturing errors that should never happen. We're making great efforts to reduce our overall Sentry volume with the intention to eventually bring it down to roughly zero, but we'll never get there if we add these sort of errors.

If something is expected to fail, but you want more information about when and how it fails, I'd suggest using a metric event instead.  Then we can track the behavior in Mixpanel.

Reply:
That sounds great, will do that for this error.